apps/ideas/views.py

```diff
@@ -94,6 +94,11 @@ class IdeaSketchCreateWizard(PermissionRequiredMixin,
     finish_section_text = _('You can add data or edit your idea later.')
     finish_section_btn = _('Submit your idea!')
 
+    def get_form_kwargs(self, step):
+        if step == '3' or step == '4':
```

Comment:
If we assume that there is only one Site we could remove this code, and simply make a DB lookup for the Site (maybe get the site id from the settings?) in the form without the request. Then we don't need to overwrite the get_form_kwargs in any of the views. Am I right? Sound like a plan.